runelite-api/src/main/java/net/runelite/api/AnimationID.java

```diff
@@ -148,9 +147,11 @@
 	public static final int FARMING_HARVEST_FLOWER = 2292;
 
 	// Lunar spellbook
+	public static final int ENERGY_TRANSFER = 4411;
 	public static final int MAGIC_LUNAR_FERTILE_SOIL = 4413;
 	public static final int MAGIC_LUNAR_CURE_PLANT = 4432;
 	public static final int MAGIC_LUNAR_GEOMANCY = 7118;
+	public static final int VENGEANCE_OTHER = 4411;
```

Comment:
Dont add duplicate animation IDs, probably just name it ENERGY_TRANSFER_VENG_OTHER or something

Reply:
Fixed